yarn.lock

```diff
@@ -1387,6 +1394,13 @@
   dependencies:
     "@types/istanbul-lib-report" "*"
 
+"@types/jquery@*":
```

Comment:
what is pulling in `@types/jquery`? does the international dropdown use jquery?

Reply:
This caught my attention as well. It's a dependency of `@types/intl-tel-input`. I think older versions of `intl-tel-input` had a dependency on / integration with jQuery, so there may be some types targeting that integration?